.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sigla-UFSCar
+    docker:
+      - image: circleci/ruby:2.4.1-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          RAILS_ENV: development
+          PGUSER: postgres
+          POSTGRES_PASSWORD: postgres
+
+      - image: circleci/postgres:9.4
+        environment:
+          POSTGRES_DB: sigla-test
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+    steps:
+      - checkout
+
+      - run:
+          name: Which bundler?
+          command: bundle -v
+
+      - run:
+          name: Install environment dependencies
+          command: sudo apt-get install -y cmake
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}
```

Comment:
What does it mean?

Reply:
We basically verify the checksum of the lock file before bundling the environment